jira/client.py

```diff
@@ -2019,6 +2020,15 @@ def votes(self, issue: str) -> Votes:
         """
         return self._find_for_resource(Votes, issue)
 
+    @translate_resource_args
+    def project_permissionscheme(self, project):
+        """Get a PermissionScheme Resource from the server.
+
+        :param project: ID or key of the project to get the permissionscheme for
+        :rtype: PermissionScheme
+        """
```

Comment:
Let's use the Google style docstring 

```suggestion

        Args:
            project (str): ID or key of the project to get the permissionscheme for

        Returns:
        PermissionScheme: The permission scheme
        """
```